### ISSUE TITLE
Server crashes during long drone pathfinding duration

### ISSUE THREAD

**Issue by pyure** [TeamPneumatic/pnc-repressurized#374]
### Minecraft Version
1.12.2

### Forge Version
forge-14.23.5.2838

### Mod Version
pneumaticcraft-repressurized-1.12.2-0.11.3-366.jar

### Describe your problem, including steps to reproduce it
Dedicated server crashes with a stacktrace complaining about Pneumaticcraft.  

### Any other comments?
Not certain Pneumaticraft is actually at fault here.  We're using a modpack where any number of mod collisions could be the real culprit.

We're not on very latest version, but changelog doesn't mention any fixes to drone pathing.

**Comment by desht** [comment]
I would need to see a stacktrace...

**Comment by ct-klein** [comment]
Hey there,  I'm from same server (it was my drone)
https://pastebin.com/QkkKr27F

**Comment by pyure** [comment]
My apologies desht.  I blame my 5yo for distracting me.  And my brain.

**Comment by desht** [comment]
No worries.  Looks like the drone pathfinder is getting into an infinite loop somehow.

**Comment by ct-klein** [comment]
Just as some background, I made a harvester drone with a simple program to harvest all pams harvestcraft plants in a 16x16 area.  No stops, just constant flying around and harvesting as the plants came up.  It was only running about 10 minutes at the time of the crash.

**Comment by desht** [comment]
Yeah, that's weird.  Any chance you could upload the program to pastebin?  (there's a button on the Programmer GUI to do that).  I can have a look and see if I can reproduce it.

**Comment by ct-klein** [comment]
It's pretty basic :)
https://pastebin.com/DxXLatJL
